src/main/java/racingcar/vo/CarListFactory.java

```diff
@@ -0,0 +1,38 @@
+package racingcar.vo;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import racingcar.model.Car;
+
+public class CarListFactory {
+    private static final String DELIMITER_CAR_NAMES = ",";
+
+    public static CarList create(final List<Car> cars) {
+        final CarList carList = new CarList(cars);
+        return carList;
+    }
+
+    public static CarList create(final String carNames) {
+        final List<String> carNameList = getCarNameListFromCarNames(carNames);
+        final List<Car> carList = getCarList(carNameList);
+        final CarList carListObject = new CarList(carList);
+        return carListObject;
+    }
```

Comment:
```suggestion
    public static CarList create(final String carNames) {
        final List<String> carNameList = getCarNameListFromCarNames(carNames);
        return getCarList(carNameList);
    }
```
이렇게 바로 리턴하면 어떨까요? 변환을 getCarList에서 해주고요!

Reply:
@skylar1220 좋은 의견 감사합니다!! 중간에 `map` 으로 변환해도 괜찮을 것 같네요!!